internal/services/session/session.go

```diff
@@ -3,38 +3,36 @@ package session
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"io"
 	"net/http"
 	"net/url"
 
 	"github.com/MicahParks/keyfunc"
-	"github.com/corbado/corbado-go/pkg/logger"
 	"github.com/golang-jwt/jwt/v4"
 	"github.com/pkg/errors"
 
+	"github.com/corbado/corbado-go/pkg/logger"
+	"github.com/corbado/corbado-go/pkg/validationerror"
+
 	"github.com/corbado/corbado-go/internal/assert"
 	entities2 "github.com/corbado/corbado-go/pkg/entities"
```

Comment:
minor: the alias entities2 is probably not needed

Reply:
yes, fixed, thx!

Reply:
done